docs/UserGuide.md

```diff
@@ -112,45 +109,97 @@ Format: `help`
 #### 4.1.2. Add a Multiple Choice Question: `mcq`
 
 Adds a multiple choice question to the question bank.
+Required Parameters:
+* `qn/` The MC Question statement
+* `opt/` The incorrect choices. 3 in total.
+* `ans/` The correct choice. 1 in total.
+* `i/` The importance of the question for the user. An integer between 1 and 3. 3 signifies the highest importance and 1 signifies the lowest importance.
+
+Optional Parameters:
+* `t/` The tags of the question 
 
 Format: `mcq qn/QUESTION opt/OPTION1 opt/OPTION2 opt/OPTION3 ans/ANSWER i/IMPORTANCE [t/TAG]`
 
+* `ANSWER` is expected only once in the command, but if you specify it multiple times, only the last occurrence will be taken.<br>
+  e.g. if you specify `ans/t ans/abc`, only `ans/abc` will be taken.
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A multiple choice question must have exactly three incorrect options and one correct answer
 </div>
 
 Examples:
-* `mcq qn/what is 1 + 1? opt/3 opt/1 opt/0 ans/2 i/1`
+* `mcq qn/What is 1 + 1? opt/3 opt/1 opt/0 ans/2 i/1`
+  * creates a new MCQ question:
+    * What is 1 + 1?
+      * A. 3
+      * B. 1
+      * C. 0
+      * D. 2 (correct)
+      
+      having importance 1.
+    
+* `mcq qn/What is 1 + 5? opt/3 opt/1 opt/0 ans/12 ans/6 i/2`
+  * creates a new MCQ question:
+    * What is 1 + 1?
+      * A. 3
+      * B. 1
+      * C. 0
+      * D. 6 (correct)
+        
+      having importance 2.
 
 #### 4.1.3. Add a True False Question: `tfq`
 
 Adds a true false question to the question bank.
 
+* `ANSWER` is expected only once in the command, but if you specify it multiple times, only the last occurrence will be taken.<br>
+  e.g. if you specify `ans/t ans/f`, only `ans/f` will be taken.
+* Answers are case-insensitive. `T` and `t` both will be accepted as `True`.
+
 Format: `tfq qn/QUESTION ans/ANSWER i/IMPORTANCE [t/TAG]`
+Required Parameters:
+* `qn/` The TF Question statement
+* `ans/` The correct choice. 1 in total, can only be either true or false.
+* `i/` The importance of the question for the user. An integer between 1 and 3. 3 signifies the highest importance and 1 signifies the lowest importance.
+
+Optional Parameters:
+* `t/` The tags of the question
```

Comment:
I feel that this is very clear. And necessary.
For consistency, I will do the same for all commands, I don't mind. It will ensure that users don't get confused.